0002-Rename-gtk3-mp-desktop-file.patch

```diff
@@ -0,0 +1,130 @@
+From fe1a5d923160d1a29b25fef9b64033c86854ca5b Mon Sep 17 00:00:00 2001
+From: Marko Lindqvist <cazfi74@gmail.com>
+Date: Sun, 14 Jan 2024 14:52:06 +0200
+Subject: [PATCH 2/3] Rename gtk3 mp desktop file
+
+Make it org.freeciv.sdl2.mp.desktop
+
+Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
+---
+ bootstrap/Makefile.am                                       | 6 +++---
+ ...k3.appdata.xml.in => org.freeciv.sdl2.mp.appdata.xml.in} | 4 ++--
+ ...iv.mp.gtk3.desktop.in => org.freeciv.sdl2.mp.desktop.in} | 0
+ configure                                                   | 4 ++--
+ configure.ac                                                | 2 +-
+ tools/Makefile.am                                           | 2 +-
+ 6 files changed, 9 insertions(+), 9 deletions(-)
+ rename bootstrap/{org.freeciv.mp.gtk3.appdata.xml.in => org.freeciv.sdl2.mp.appdata.xml.in} (94%)
+ rename bootstrap/{org.freeciv.mp.gtk3.desktop.in => org.freeciv.sdl2.mp.desktop.in} (100%)
+
+diff --git a/bootstrap/Makefile.am b/bootstrap/Makefile.am
+index 85aeb4d548..507d041a77 100644
+--- a/bootstrap/Makefile.am
++++ b/bootstrap/Makefile.am
+@@ -27,7 +27,7 @@ EXTRA_DIST =	freeciv.project		\
+ 		org.freeciv.qt.desktop.in	\
+ 		org.freeciv.server.desktop.in	\
+ 		org.freeciv.mp.gtk2.desktop.in	\
+-		org.freeciv.mp.gtk3.desktop.in	\
++		org.freeciv.sdl2.mp.desktop.in	\
+ 		org.freeciv.mp.gtk4.desktop.in	\
+ 		org.freeciv.mp.qt.desktop.in	\
+ 		org.freeciv.ruledit.desktop.in	\
+@@ -38,7 +38,7 @@ EXTRA_DIST =	freeciv.project		\
+ 		org.freeciv.qt.appdata.xml.in \
+ 		org.freeciv.server.appdata.xml.in \
+ 		org.freeciv.mp.gtk2.appdata.xml.in \
+-		org.freeciv.mp.gtk3.appdata.xml.in \
++		org.freeciv.sdl2.mp.appdata.xml.in \
+ 		org.freeciv.mp.gtk4.appdata.xml.in \
+ 		org.freeciv.mp.qt.appdata.xml.in \
+ 		org.freeciv.ruledit.appdata.xml.in \
+@@ -77,7 +77,7 @@ metainfofile_DATA += org.freeciv.mp.gtk2.appdata.xml
+ endif
+ 
+ if MODINST_GTK3
+-metainfofile_DATA += org.freeciv.mp.gtk3.appdata.xml
++metainfofile_DATA += org.freeciv.sdl2.mp.appdata.xml
+ endif
+ 
+ if MODINST_GTK4
+diff --git a/bootstrap/org.freeciv.mp.gtk3.appdata.xml.in b/bootstrap/org.freeciv.sdl2.mp.appdata.xml.in
+similarity index 94%
+rename from bootstrap/org.freeciv.mp.gtk3.appdata.xml.in
+rename to bootstrap/org.freeciv.sdl2.mp.appdata.xml.in
+index 7a7ad972a3..df93e1ebde 100644
+--- a/bootstrap/org.freeciv.mp.gtk3.appdata.xml.in
++++ b/bootstrap/org.freeciv.sdl2.mp.appdata.xml.in
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop-application">
+     <name>Freeciv gtk+-3 modpack installer</name>
+-    <id>org.freeciv.mp.gtk3</id>
++    <id>org.freeciv.sdl2.mp</id>
```

Comment:
whatever you are doing this doesn't match the app-id in the manifest. The result will be a build failure.

Reply:
oh I see. it will just be ignored. There is only one appstream file installed.

Reply:
The org.freeciv.sdl2 part does match app-id, and with additional ".mp" it's creating a secondary launcher. Builds fine, and works as expected in my tests.

Reply:
https://stackoverflow.com/questions/71502679/is-it-possible-to-make-multiple-commands-from-one-flatpak-app